content/install-hook-openusercss.js

```diff
@@ -0,0 +1,159 @@
+'use strict';
+
+const manifest = chrome.runtime.getManifest();
```

Comment:
Wrap the entire code in IIFE because content scripts may be reinjected on update or on manual reload and there are problems with global const/let at least in some browser versions. See install-hook-userstyles.js on how to check for the orphaned state.

Reply:
install-hook-usercss.js might be a better example of orphanCheck